contrib/pkg/createcluster/create.go

```diff
@@ -180,6 +180,11 @@ type Options struct {
 	AzureCloudName                   string
 	AzureResourceGroupName           string
 
+	// GCP
+	DiscardLocalSsdOnHibernate bool
+	// And because we only want to pass this through if explicitly provided by the user...
+	discardLocalSsdOnHibernate *bool
+
```

Comment:
Uh, why is the private var necessary? Or maybe an explanatory note on how they differ would help here, or at the comment on Line 504